app/utils/progress-bar-utils.ts

```diff
@@ -17,7 +17,8 @@ function buildProgressBarStages(state: string) {
         States.AWAITING_CLARIFYING_QUESTIONS.id,
         States.CLARIFYING_QUESTIONS_SUBMITTED.id,
         States.AWAITING_CMA_REQUIREMENTS.id,
-        States.CMA_REQUIREMENTS_SUBMITTED.id
+        States.CMA_REQUIREMENTS_SUBMITTED.id,
```

Comment:
Between CMA_REQUIREMENTS_SUBMITTED and CMA_LISTED.
There's also another state CMA_ADJUSTMENTS_AGREED, we need to ensure that we map this state as well to the correct progress bar and do this next section.